Source/DafnyLanguageServer/Workspace/ChangeProcessors/Relocator.cs

```diff
@@ -139,8 +138,6 @@ CancellationToken cancellationToken
         var newRelatedInformation = diagnostic.RelatedInformation?.SelectMany(related =>
           MigrateRelatedInformation(change, related)).ToList();
         yield return diagnostic with {
-          Message = diagnostic.Message.StartsWith(OutdatedPrefix) ? diagnostic.Message : OutdatedPrefix + diagnostic.Message,
```

Comment:
I haven't tested it, but is it actually removing the "outdated" messages of verification? I think i liked them a lot.

Reply:
The code is just moved.